ci/cppcheck/cppcheck_existing_issues.txt

```diff
@@ -1,399 +1,762 @@
-'/builds/prpl-foundation/prplMesh/framework/tlvf/test/tlvf_test.cpp: style: Variable 'tlv3' is assigned a value that is never used. [unreadVariable]    auto tlv3 = msg.addClass<tlvWsc>();
-'/builds/prpl-foundation/prplMesh/controller/src/beerocks/master/son_management.cpp: style: Variable 'result_status' is assigned a value that is never used. [unreadVariable]        auto result_status = eChannelScanErrCode::CHANNEL_SCAN_SUCCESS;
-'/builds/prpl-foundation/prplMesh/controller/src/beerocks/master/son_management.cpp: style: Variable 'op_error_code' is assigned a value that is never used. [unreadVariable]        auto op_error_code = eChannelScanOpErrCode::CHANNEL_SCAN_OP_SUCCESS;
-'/builds/prpl-foundation/prplMesh/controller/src/beerocks/master/son_management.cpp: style: Variable 'op_error_code' is assigned a value that is never used. [unreadVariable]            op_error_code = eChannelScanOpErrCode::CHANNEL_SCAN_OP_SCAN_IN_PROGRESS;
-'/builds/prpl-foundation/prplMesh/controller/src/beerocks/master/son_management.cpp: style: Variable 'result_status' is assigned a value that is never used. [unreadVariable]            result_status = last_scan_success;
-'/builds/prpl-foundation/prplMesh/framework/tlvf/AutoGenerated/src/tlvf/wfa_map/tlvAssociatedStaLinkMetrics.cpp: style: The if condition is the same as the previous if condition [duplicateCondition]    if (m_parse__) {
-'/builds/prpl-foundation/prplMesh/framework/tlvf/AutoGenerated/src/tlvf/wfa_map/tlvAssociatedStaLinkMetrics.cpp: style: C-style pointer casting [cstyleCast]    m_bssid_info_list = (sBssidInfo*)m_buff_ptr__;
-'/builds/prpl-foundation/prplMesh/framework/tlvf/AutoGenerated/src/tlvf/wfa_map/tlvStaMacAddressType.cpp: style: The if condition is the same as the previous if condition [duplicateCondition]    if (m_parse__) {
-'/builds/prpl-foundation/prplMesh/framework/tlvf/AutoGenerated/src/tlvf/wfa_map/tlvMetricReportingPolicy.cpp: style: The if condition is the same as the previous if condition [duplicateCondition]    if (m_parse__) {
-'/builds/prpl-foundation/prplMesh/framework/tlvf/AutoGenerated/src/tlvf/wfa_map/tlvMetricReportingPolicy.cpp: style: C-style pointer casting [cstyleCast]    m_metrics_reporting_conf_list = (sMetricsReportingConf*)m_buff_ptr__;
-'/builds/prpl-foundation/prplMesh/framework/tlvf/AutoGenerated/src/tlvf/wfa_map/tlvSteeringPolicy.cpp: style: The if condition is the same as the previous if condition [duplicateCondition]    if (m_parse__) {
-'/builds/prpl-foundation/prplMesh/framework/tlvf/AutoGenerated/src/tlvf/wfa_map/tlvSteeringPolicy.cpp: style: C-style pointer casting [cstyleCast]    m_radio_ap_control_policy_list = (sRadioApControlPolicy*)m_buff_ptr__;
+'common/beerocks/bcl/include/bcl/network/socket.h: style: Statements following return, break, continue, goto or throw will never be executed. [unreachableCode]        m_error.clear();
+        ^'
```

Comment:
There seems to be some commit reordering going on here, because the carets are re-introduced here...